docs/source/contributing/index.md

```diff
@@ -50,11 +50,57 @@ The Volto Team reviews pull requests only from people with a GitHub account who
 
 (contributing-translations-label)=
 
+## Install Volto for development
+
+To make changes to Volto, you need to run it from a copy of the `plone/volto` repository.
+
+### Prerequisites
+
+You need all the requirements already mentioned in {doc}`plone:install-packages-prerequisites-label`.
+
+### Clone the Volto repository
+
+```shell
+git clone https://github.com/plone/volto.git
+```
+
+### Start the Plone backend
+
+While developing Volto, you need to have the Plone backend running.
+If you don't already have the backend installed,
+the easiest way is to run the following command inside the Volto repository:
+
+```shell
+make start-backend-docker
+```
+
+### Install Node.js dependencies
+
+```shell
+yarn
+```
+
+### Start Volto
+
+```
+yarn start
+```
+
+### Open Volto in your browser
+
+Browse to [http://localhost:3000](http://localhost:3000).
+
 ## Translations
 
 All text that can be shown in a browser must be translatable.
 Please mark all such strings as translatable as defined in the [i18n guide](../recipes/i18n.md).
 
+### Branch policy for translations
+
+Due to the nature of `main` and `16.x.x` branches, some developments that may land in `main` may not be backported to `16.x.x`. This means that many translations that may come with those developments will be useless in the `16.x.x` branch and thus porting them to `16.x.x` makes no sense.
+
+So when contributing translations, please create PRs directly from branches created from `16.x.x` and point your PRs to that exact branch instead of `main`.
```

Comment:
```suggestion
When contributing translations, please create pull requests directly from branches created from `16.x.x`, and point your pull requests to that exact branch instead of `main`.
```